test-util/src/main/java/org/jboss/windup/testutil/html/TestJavaApplicationOverviewUtil.java

```diff
@@ -77,6 +77,7 @@ public void checkMainEffort(int expectedEffort)
                                 ".//div[contains(@class, 'container') and contains(@class, 'mainGraphContainer')]//div[@class = 'points']/div[@class = 'number']"));
         String effortString = effortElement.getText().trim();
         effortString = effortString.replace(",", "");
+        effortString = effortString.replace(".", "");
```

Comment:
This seems to me like you have set up different locale environment than UK/US for number delimiters. We have on Jenkins CI for master branch and this failure doesn't show up there.